utils/misc.mli

```diff
@@ -118,8 +139,9 @@ val replace_substring: before:string -> after:string -> string -> string
            occurences of [before] with [after] in [str] and returns
            the resulting string. *)
 
-val rev_split_words: string -> string list
-        (* [rev_split_words s] splits [s] in blank-separated words, and return
+val rev_split_words: ?separator:char -> string -> string list
```

Comment:
But the default value in this case is not one single character, it is a more general notion of "blank space" that spans multiple characters. This underlines the fact that either your separate API is inadequate, or those are really two different functions in hiding.